components/timezone/src/ixdtf.rs

```diff
@@ -78,116 +78,214 @@ impl UtcOffset {
-                };
-
-                Ok(Self {
-                    time_zone_id,
-                    offset,
-                    zone_variant,
-                    local_time,
-                })
-            }
-            TimeZoneRecord::Offset(offset_record) => {
-                let tz = Self::try_from_utc_offset_record(offset_record)?;
-                if offset.is_some() && tz.offset != offset {
-                    return Err(ParseError::InconsistentTimeZoneOffsets);
                 }
-                Ok(tz)
-            }
-            _ => Err(ParseError::MissingTimeZone),
-        }
+                None => {
+                    // time_zone_id not found; Etc/Unknown?
+                    debug_assert_eq!(time_zone_id.0.as_str(), "unk");
+                    ZoneVariant::standard()
+                }
+            };
+        Ok(Self {
+            time_zone_id,
+            offset: Some(offset),
+            zone_variant,
+            local_time: (iso.date, iso.time),
+        })
     }
 }
 
 // ==== CustomZonedDateTime methods and traits ====
 
-impl CustomZonedDateTime<Iso> {
+impl CustomZonedDateTime<Iso, models::Full> {
+    /// Create a [`CustomZonedDateTime`] in ISO-8601 calendar from an IXDTF syntax string.
+    ///
+    /// The string should have only an offset and no named time zone.
+    ///
+    /// ✨ *Enabled with the `compiled_data` and `ixdtf` Cargo features.*
+    pub fn try_offset_only_iso_from_str(ixdtf_str: &str) -> Result<Self, ParseError> {
+        Self::try_offset_only_iso_from_utf8(ixdtf_str.as_bytes())
+    }
+
+    /// Create a [`CustomZonedDateTime`] in ISO-8601 calendar from IXDTF syntax utf8 bytes.
+    ///
+    /// The string should have only an offset and no named time zone.
+    ///
+    /// ✨ *Enabled with the `compiled_data` and `ixdtf` Cargo features.*
+    ///
+    /// See [`Self::try_iso_from_str`].
+    pub fn try_offset_only_iso_from_utf8(ixdtf_str: &[u8]) -> Result<Self, ParseError> {
+        let ixdtf_record = IxdtfParser::from_utf8(ixdtf_str).parse()?;
+        let time_zone = TimeZoneInfo::try_offset_only_from_ixdtf_record(&ixdtf_record)?;
+        Self::try_iso_from_ixdtf_record(&ixdtf_record, time_zone)
+    }
+}
+
+impl CustomZonedDateTime<Iso, models::AtTime> {
     /// Create a [`CustomZonedDateTime`] in ISO-8601 calendar from an IXDTF syntax string.
     ///
+    /// The string should have only a named time zone and no offset.
+    ///
+    /// ✨ *Enabled with the `compiled_data` and `ixdtf` Cargo features.*
+    pub fn try_location_only_iso_from_str(ixdtf_str: &str) -> Result<Self, ParseError> {
+        Self::try_location_only_iso_from_utf8(ixdtf_str.as_bytes())
+    }
+
+    /// Create a [`CustomZonedDateTime`] in ISO-8601 calendar from IXDTF syntax utf8 bytes.
+    ///
+    /// The string should have only a named time zone and no offset.
+    ///
+    /// ✨ *Enabled with the `compiled_data` and `ixdtf` Cargo features.*
+    ///
+    /// See [`Self::try_iso_from_str`].
```

Comment:
I think we usually just say "See `Self::try_from_str`" to avoid overloading the docs pages with repeated information.

Reply:
Hmm, maybe we should discuss that. I think critical information should be duplicated. For example, it is not useful to see a tooltip that says "see this other thing". More detailed docs and examples should be deduplicated, sure.